File: Belah Ketupat (Fill)/C#/belah-ketupat-i.cs
using System;
class belahKetupat {
  static void Main() {
    Console.Write("Masukan Lebar Belah Ketupat : ");
    int x = int.Parse(Console.ReadLine());
    for(int i = 1; i <= x; i++){
        for(int j = x; j >= i; j--){
            Console.Write(" ");
        }
        
        for(int l = 2; l <= i; l++){
            Console.Write("*");
        }
        
        for(int k = 1; k <= i; k++){
                Console.Write("*");
            }
        Console.Write("\n");
    }
    
    for(int i = 1; i <= (x - 1); i++){
        for(int j = x; j <= (i + x); j++){
            Console.Write(" ");
        }
        
        for(int l = 0; l <= (x - i - 1); l++){
            Console.Write("*");
        }
        
        for(int k = 0; k <= x-i-2; k++){
                Console.Write("*");
            }
        Console.Write("\n");
    }
  }
}

File: Belah Ketupat (Stroke)/Java/belah-ketupat-i.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args){
	    
	    Scanner userInput = new Scanner(System.in);
        
        System.out.print("Masukan Lebar Belah Ketupat: ");
        int x = userInput.nextInt();
      
        for(int h = 1; h <= x + 1; h++){
            System.out.print(" ");
        }
       
        System.out.print("*\n");
       
        for(int i = 1; i <= x; i++){
            for(int j = x; j >= i; j--){
                System.out.print(" ");
            }
        
            for(int l = 1; l <= 1; l++){
                System.out.print("*");
            }
            
            for(int k = 1; k <= i; k++){
                System.out.print(" ");
            }
        
            for(int m = 2; m <= i; m++){
                System.out.print(" ");
            }  
            
            System.out.print("*\n");
        }
    
        System.out.print("*");
    
        for(int v = 0; v <= x*2; v++){
            System.out.print(" ");
        }
    
        System.out.print("*\n");
    
        for(int p = 0; p <= x-1; p++){
            for(int q = 0; q <= p; q++){
                System.out.print(" ");
            }
           
            System.out.print("*");
           
            for(int s = x-1; s >= p; s--){
               System.out.print(" ");
            }
          
            for(int t = x-2; t >= p; t--){
                System.out.print(" ");
            }  
          
            System.out.print("*\n");
        }
        
        for(int u = 1; u <= x+1; u++){
            System.out.print(" ");
        }
    
        System.out.print("*");
	}
}
